crafters/image/ImageReader/manifest.yml

```diff
@@ -7,6 +7,6 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub/blob/master/crafters/image/ImageReader/README.md
-version: 0.0.14
```

Comment:
image reader version does not have to be updated. please update the image normalizer version instead

Reply:
good catch!